packages/xchain-ethereum/__mocks__/thornode-api.ts

```diff
@@ -10,6 +10,12 @@ type InboundAddress = {
   router?: string
 }
 
+type MimirDetails = {
+  HALTTRADING: number
+  HALTETHTRADING: number
+  HALTETHCHAIN: number
+}
```

Comment:
Can you define type of `MimirDetails` in `xcchain-util` to use it here and everywhere else, pls?

Reply:
But this type does not carry anything useful. It created because ESLint disallow use `any` type and mimir details has different params for mainnet and testnet and as I understood it does`nt have "standart" variables.